### PR TITLE
Feature/automation 16341 generate pin list ui

### PR DESCRIPTION
**Pull request by sinhaamit1980**

### CODE REVIEW

File: admin/test/features/generate_pupil_pins.feature [hunk 1]
```diff
@@ -2,10 +2,34 @@
...
+  | teacher2 |
+  | teacher3 |
+  | teacher4 |
+
+  Scenario: Generate Pupil List Page do not display pupil with active pin
+    Given I am logged in
+    And I have a pupil with active pin
+    And I am on the generate pupil pins page
+    When I click Generate PINs button
+    Then I CANNOT see this pupil in the list of Pupil on Generate Pin list page
```
Comment (activemq) [inline admin/test/features/generate_pupil_pins.feature]: @sinhaamit1980 is there a reason for the capitals?

File: admin/test/features/generate_pupil_pins.feature [hunk 2]
```diff
@@ -2,10 +2,34 @@
...
+    Then I should see a list of pupils sorted by surname on Generate Pins List Page
+
+  Examples:
+  | teacher  |
+  | teacher1 |
+  | teacher2 |
+  | teacher3 |
+  | teacher4 |
+
+  Scenario: Generate Pupil List Page do not display pupil with active pin
```
Comment (activemq) [inline admin/test/features/generate_pupil_pins.feature]: @sinhaamit1980  does this page actually exist? "Generate Pupil List Page" ??

Comment (sinhaamit1980) [reply admin/test/features/generate_pupil_pins.feature]: Done changed it to Generate Pins Pupil List page

File: admin/test/features/generate_pupil_pins.feature (hunk 1 above)
Comment (sinhaamit1980) [reply admin/test/features/generate_pupil_pins.feature]: Changed to Lowercase

**Pull request merged**